src/app/components/crypto-assets/crypto-currency-asset/crypto-currency-asset-item.tsx

```diff
@@ -58,11 +59,19 @@ export const CryptoCurrencyAssetItem = forwardRefWithAs(
       onCopy();
       toast.success('Address copied!');
     }
+
+    // copy icon is needed only if asset address can be copied
+    let copyIcon;
+    if (canCopy) {
+      copyIcon = AssetItemCopyIcon({ hasCopied });
+    }
+
     return (
       <CryptoCurrencyAssetItemLayout
         balance={balance}
         caption={assetBalance.balance.symbol}
         icon={icon}
+        copyIcon={copyIcon}
```

Comment:
```suggestion
        copyIcon={canCopy ? <AssetItemCopyIcon hasCopied={hasCopied} /> : null}
```

the mutatative code above looks funky, isn't this simpler? In general, imo it's best to keep JSX as JSX and not call as a fn like `AssetItemCopyIcon({ hasCopied })`

Reply:
yes, agree here, thanks